core/src/main/scala/org/apache/spark/deploy/history/ApplicationCache.scala

```diff
@@ -156,18 +156,18 @@ private[history] class ApplicationCache(
    */
   @throws[NoSuchElementException]
   private def loadApplicationEntry(appId: String, attemptId: Option[String]): CacheEntry = {
-    logDebug(s"Loading application Entry $appId/$attemptId")
+    logDebug(s"Loading application Entry $appId/${attemptId.mkString}")
```

Comment:
Let's put `$appId/${attemptId.mkString}` to a lazy val
```
lazy val application = s"$appId/${attemptId.mkString}"
```
to reduce duplciate code.